monkey/infection_monkey/exploit/vsftpd.py

```diff
@@ -0,0 +1,142 @@
+import StringIO
```

Comment:
Please add a few lines describing your exploit. For example, see `elasticgroovy.py` or `hadoop.py`

Reply:
Done.